.eslintrc.js

```diff
@@ -43,13 +37,36 @@ module.exports = {
         '@typescript-eslint/no-object-literal-type-assertion': 'off',
         '@typescript-eslint/no-use-before-define': 'off',
         '@typescript-eslint/explicit-module-boundary-types': 'off',
-        '@typescript-eslint/no-duplicate-imports': 'error',
+        '@typescript-eslint/no-duplicate-imports': 'warn',
+        '@typescript-eslint/array-type': ['warn', {
```

Comment:
I find it more suitable to be `string[]`, or `MyType[]` or `Promise<MyType[]>` than `Array<string>`, `Array<MyType>` and `Promise<Array<MyType>>`. But something like `(() => void)[]` can look strange. Let's try a magic [`array-simple` option](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/array-type.md#array-simple).